src/components/Composer/index.native.js

```diff
@@ -90,6 +126,21 @@ class Composer extends React.Component {
         this.props.onClear();
     }
 
+    onChange(event) {
+        this.mostRecentEventCount = event.nativeEvent.eventCount;
+        this.props.onChange(event);
+    }
+
+    setTextAndSelection(text, start, end) {
+        this.viewCommands.setTextAndSelection(
+            this.textInput,
+            this.mostRecentEventCount,
+            text,
+            start,
+            end,
+        );
+    }
```

Comment:
The only differences to the `setSelection` exposed on the textInput ref is that the built-in `setSelection` seem to pass `null` for `text`: 

https://github.com/facebook/react-native/blob/f353119113d6fc85491765ba1e90ac83cb00fd61/Libraries/Components/TextInput/TextInput.js#L1196-L1206

Reply:
nw, i fall back to using `value` prop and the existing `setSelection` method